src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -597,6 +599,9 @@ private void recursivelyCopyProjectDependenciesWithScope(
                 GradleWorkarounds.fixAttributesOfModuleDependency(projectDep.getObjects(), externalDep);
                 externalDep.attributes(attr -> attr.attribute(GCV_SCOPE_ATTRIBUTE, scope));
             });
+            // To avoid capability based conflict detection between all these copied configurations, we give each
+            // of them a totally random capability
+            copiedConf.getOutgoing().capability(String.format("gcv:%s:0", UUID.randomUUID().toString()));
```

Comment:
could we make this deterministic (e.g. a hash) just to avoid any danger of it affecting reproducibility / caching?